src/app/templates/_bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "<%= nameOf.package %>",
+  "version": "0.0.0",
```

Comment:
We should follow `npm`s default of 1.0.0, I think. (Though, it does make it weird trying to version and publish your first release).